xc/doc/man/general/Consortium.man
.\" $XConsortium: Consortium.man /main/70 1996/12/18 09:51:36 rws $
.\" Copyright (c) 1993, 1994, 1996  X Consortium
.\" 
.\" Permission is hereby granted, free of charge, to any person obtaining a
.\" copy of this software and associated documentation files (the "Software"), 
.\" to deal in the Software without restriction, including without limitation 
.\" the rights to use, copy, modify, merge, publish, distribute, sublicense, 
.\" and/or sell copies of the Software, and to permit persons to whom the 
.\" Software furnished to do so, subject to the following conditions:
.\" 
.\" The above copyright notice and this permission notice shall be included in
.\" all copies or substantial portions of the Software.
.\" 
.\" THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
.\" IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
.\" FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL 
.\" THE X CONSORTIUM BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, 
.\" WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF 
.\" OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE 
.\" SOFTWARE.
.\" 
.\" Except as contained in this notice, the name of the X Consortium shall not 
.\" be used in advertising or otherwise to promote the sale, use or other 
.\" dealing in this Software without prior written authorization from the 
.\" X Consortium.
.TH XCONSORTIUM 7 "Release 6.3"  "X Version 11"
.SH NAME
XConsortium \- X Consortium information
.SH SYNOPSIS
Release 6.3 of X Version 11 was brought to you by X Consortium, Inc.
.SH DESCRIPTION
The X Consortium was an independent, not-for-profit Delaware membership
corporation.  It was formed in 1993 as the successor to the MIT X Consortium.
The purpose of the X Consortium was to foster the development, evolution, and
maintenance of the X Window System, a comprehensive set of vendor-neutral,
system-architecture neutral, network-transparent windowing and user interface
standards.
.PP
The X Window System was created in the mid-1980s at the Massachusetts
Institute of Technology.  In 1988, MIT formed a member-funded consortium to
provide the technical and administrative leadership necessary to support
further development of the X Window System.  In 1992, MIT and the membership
decided it was in their best interests to move the consortium out of MIT and
create an independent, stand-alone organization.  All rights to the
X Window System were assigned by MIT to X Consortium, Inc. on January 1, 1994.
X Consortium, Inc. closed its doors on December 31, 1996.  All rights to the
X Window System have been assigned to the Open Software Foundation.
.PP
The X Consortium was financially self-supporting through membership fees.
There are no license fees associated with the use of X Window System standards
and code developed by the X Consortium.  Membership in the X Consortium was
open to any organization willing to execute a membership agreement.
.PP
The X Consortium was a highly participative body.  Members were encouraged to
actively cooperate with the staff and other members in the design and review
of proposed specifications, and in the design, coding and testing of sample
implementations of those specifications.
.PP
The X Consortium accomplished most of its work using electronic mail over the
Internet, with individual mailing lists for working groups.  Internet
electronic mail connectivity was viewed as a requirement for useful
participation in X Consortium activities.  Meetings were held as necessary,
often in conjunction with industry conferences and trade shows.
.SH STAFF
.nf
President:
Bob Scheifler

Office Manager:
Janet O'Halloran

Director of Marketing:
Paul Lavallee

Director of Engineering:
Jim Fournier

Manager, X Window System:
Matt Landau, emeritus

Technical Director, X Window System:
Ralph Swick

Technical Staff, X Window System:

Donna Converse, emeritus
Stephen Gildea, emeritus
Kaleb Keithley
Arnaud Le Hors
Ralph Mor, emeritus
Ray Tice
Dave Wiggins, emeritus

Managers, CDE Development:
Giora Guth
Peter Bohnert, emeritus

Manager, CDE Quality Engineering:
David Brooks

CDE Architects:
Kevin Samborn
Daniel Dardailler, emeritus

Technical Staff, CDE Development:

Art Barstow
Pascale Dardailler
David Kaelbling
Mitch Greess
Robert Seacord

Technical Staff, CDE Quality Engineering:

Chris Burleson
Tom Cavin
Sami Mohammed
Mark Schuldenfrei

Manager, Systems Administration:
Kevin Ethier

Technical Staff, Systems Administration:
Mike Donati
Amy Rich, emeritus
Anne Salemme
.fi

.SH "BOARD OF DIRECTORS"
The X Consortium's activities and affairs were managed under the direction and
oversight of a Board of Directors, elected annually by the Members.  The Board
was responsible for reviewing the achievements of the Consortium, approving
planned work, appointing a President and other officers of the Consortium, and
setting membership dues.  The last Directors were:

.nf
Robert W. Scheifler, President, X Consortium
Dr. Forest Baskett, Senior VP of R&D, Silicon Graphics Computer Systems
Harold D. Blair, Apogee International Corp.
Roger S. Gourd, Gourd & Associates
Dr. Robin Hillyard, Chairman and Chief Technical Officer, Novasoft Systems
Don McGovern, General Operations Manager and Executive Dir., Hewlett Packard
Peter J. Shaw, Senior VP, NetManage
Michael Tobias, President, Tech-Source, Inc.
.fi

.SH "ADDRESS"
To reach the X Consortium public Wide World Web server, use the URL:
http://www.x.org/
.PP
To reach the X Consortium public ftp machine, use anonymous ftp to:
ftp.x.org

.SH "FULL MEMBERS"

.nf
Adobe Systems Inc.
Cray Research, Inc.
Digital Equipment Corp.
Fujitsu Limited
Hewlett-Packard Company
Hitachi Ltd.
IBM Corporation
Megatek Corp.
Motorola, Inc.
NEC Corporation
Novell, Inc.
Oki Electric Industry Co., Ltd.
OMRON Corporation
SCO, Inc.
Siemens Nixdorf Informationssysteme AG
Silicon Graphics, Inc.
Sony Corporation
Sun Microsystems, Inc.
Tektronix, Inc.
.fi

.SH "ASSOCIATE MEMBERS"

.nf
Boundless Technologies
Hummingbird Communications Ltd.
Insignia Solutions, Ltd.
Mercury Interactive Corp.
NetManage, Inc.
Network Computing Devices
VisiCom Laboratories, Inc.
Walker Richer & Quinn, Inc.
.fi

.SH "END USERS"

.nf
Hughes Aircraft Company
.fi

.SH "AFFILIATE MEMBERS"

.nf
ASTEC, Inc.
BARCO Chromatics, Inc.
CenterLine Software, Inc.
CliniComp, Intl.
Component Integration Laboratories, Inc.
Draper Laboratory.
Electronic Book Technologies, Inc.
Gallium Software, Inc.
Georgia Institiute of Technology
Human Designed Systems, Inc.
INRIA \- Institut National de Recherche en Informatique et en Automatique
Integrated Computer Solutions, Inc.
Investment Management Services, Inc.
Jupiter Systems
KL Group Inc.
Massachusetts Institute of Technology
Metheus Corporation
Metro Link, Inc.
Object Management Group, Inc.
Open Software Foundation
Performance Awareness Corp.
Peritek Corp.
Petrotechnical Open Software Corp.
Point Technologies, Inc.
Shiman Associates, Inc.
Smithsonian Astrophysical Observatory.
Software Development Corp.
SOUM Corporation
Spectragraphics Corp.
Tech-Source, Inc.
TriTeal Corp.
White Pine Software, Inc.
World Wide Web Consortium.
The XFree86 Project, Inc.
X Inside, Inc.
.fi


xfree/xc/doc/man/general/Consortium.cpp
.\" $TOG: Consortium.cpp /main/71 1997/10/13 14:55:16 kaleb $
.\" Copyright (c) 1993, 1994, 1996  X Consortium
.\" 
.\" Permission is hereby granted, free of charge, to any person obtaining a
.\" copy of this software and associated documentation files (the "Software"), 
.\" to deal in the Software without restriction, including without limitation 
.\" the rights to use, copy, modify, merge, publish, distribute, sublicense, 
.\" and/or sell copies of the Software, and to permit persons to whom the 
.\" Software furnished to do so, subject to the following conditions:
.\" 
.\" The above copyright notice and this permission notice shall be included in
.\" all copies or substantial portions of the Software.
.\" 
.\" THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
.\" IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
.\" FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL 
.\" THE X CONSORTIUM BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, 
.\" WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF 
.\" OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE 
.\" SOFTWARE.
.\" 
.\" Except as contained in this notice, the name of the X Consortium shall not 
.\" be used in advertising or otherwise to promote the sale, use or other 
.\" dealing in this Software without prior written authorization from the 
.\" X Consortium.
.TH XCONSORTIUM __miscmansuffix__ "Release 6.3"  "X Version 11"
.SH NAME
XConsortium \- X Consortium information
.SH SYNOPSIS
Release 6.3 of X Version 11 was brought to you by X Consortium, Inc.
.SH DESCRIPTION
The X Consortium was an independent, not-for-profit Delaware membership
corporation.  It was formed in 1993 as the successor to the MIT X Consortium.
The purpose of the X Consortium was to foster the development, evolution, and
maintenance of the X Window System, a comprehensive set of vendor-neutral,
system-architecture neutral, network-transparent windowing and user interface
standards.
.PP
The X Window System was created in the mid-1980s at the Massachusetts
Institute of Technology.  In 1988, MIT formed a member-funded consortium to
provide the technical and administrative leadership necessary to support
further development of the X Window System.  In 1992, MIT and the membership
decided it was in their best interests to move the consortium out of MIT and
create an independent, stand-alone organization.  All rights to the
X Window System were assigned by MIT to X Consortium, Inc. on January 1, 1994.
X Consortium, Inc. closed its doors on December 31, 1996.  All rights to the
X Window System have been assigned to the Open Software Foundation.
.PP
The X Consortium was financially self-supporting through membership fees.
There are no license fees associated with the use of X Window System standards
and code developed by the X Consortium.  Membership in the X Consortium was
open to any organization willing to execute a membership agreement.
.PP
The X Consortium was a highly participative body.  Members were encouraged to
actively cooperate with the staff and other members in the design and review
of proposed specifications, and in the design, coding and testing of sample
implementations of those specifications.
.PP
The X Consortium accomplished most of its work using electronic mail over the
Internet, with individual mailing lists for working groups.  Internet
electronic mail connectivity was viewed as a requirement for useful
participation in X Consortium activities.  Meetings were held as necessary,
often in conjunction with industry conferences and trade shows.
.SH STAFF
.nf
President:
Bob Scheifler

Office Manager:
Janet O'Halloran

Director of Marketing:
Paul Lavallee

Director of Engineering:
Jim Fournier

Manager, X Window System:
Matt Landau, emeritus

Technical Director, X Window System:
Ralph Swick

Technical Staff, X Window System:

Donna Converse, emeritus
Stephen Gildea, emeritus
Kaleb Keithley
Arnaud Le Hors
Ralph Mor, emeritus
Ray Tice
Dave Wiggins, emeritus

Managers, CDE Development:
Giora Guth
Peter Bohnert, emeritus

Manager, CDE Quality Engineering:
David Brooks

CDE Architects:
Kevin Samborn
Daniel Dardailler, emeritus

Technical Staff, CDE Development:

Art Barstow
Pascale Dardailler
David Kaelbling
Mitch Greess
Robert Seacord

Technical Staff, CDE Quality Engineering:

Chris Burleson
Tom Cavin
Sami Mohammed
Mark Schuldenfrei

Manager, Systems Administration:
Kevin Ethier

Technical Staff, Systems Administration:
Mike Donati
Amy Rich, emeritus
Anne Salemme
.fi

.SH "BOARD OF DIRECTORS"
The X Consortium's activities and affairs were managed under the direction and
oversight of a Board of Directors, elected annually by the Members.  The Board
was responsible for reviewing the achievements of the Consortium, approving
planned work, appointing a President and other officers of the Consortium, and
setting membership dues.  The last Directors were:

.nf
Robert W. Scheifler, President, X Consortium
Dr. Forest Baskett, Senior VP of R&D, Silicon Graphics Computer Systems
Harold D. Blair, Apogee International Corp.
Roger S. Gourd, Gourd & Associates
Dr. Robin Hillyard, Chairman and Chief Technical Officer, Novasoft Systems
Don McGovern, General Operations Manager and Executive Dir., Hewlett Packard
Peter J. Shaw, Senior VP, NetManage
Michael Tobias, President, Tech-Source, Inc.
.fi

.SH "ADDRESS"
To reach the X Consortium public Wide World Web server, use the URL:
http://www.x.org/
.PP
To reach the X Consortium public ftp machine, use anonymous ftp to:
ftp.x.org

.SH "FULL MEMBERS"

.nf
Adobe Systems Inc.
Cray Research, Inc.
Digital Equipment Corp.
Fujitsu Limited
Hewlett-Packard Company
Hitachi Ltd.
IBM Corporation
Megatek Corp.
Motorola, Inc.
NEC Corporation
Novell, Inc.
Oki Electric Industry Co., Ltd.
OMRON Corporation
SCO, Inc.
Siemens Nixdorf Informationssysteme AG
Silicon Graphics, Inc.
Sony Corporation
Sun Microsystems, Inc.
Tektronix, Inc.
.fi

.SH "ASSOCIATE MEMBERS"

.nf
Boundless Technologies
Hummingbird Communications Ltd.
Insignia Solutions, Ltd.
Mercury Interactive Corp.
NetManage, Inc.
Network Computing Devices
VisiCom Laboratories, Inc.
Walker Richer & Quinn, Inc.
.fi

.SH "END USERS"

.nf
Hughes Aircraft Company
.fi

.SH "AFFILIATE MEMBERS"

.nf
ASTEC, Inc.
BARCO Chromatics, Inc.
CenterLine Software, Inc.
CliniComp, Intl.
Component Integration Laboratories, Inc.
Draper Laboratory.
Electronic Book Technologies, Inc.
Gallium Software, Inc.
Georgia Institiute of Technology
Human Designed Systems, Inc.
INRIA \- Institut National de Recherche en Informatique et en Automatique
Integrated Computer Solutions, Inc.
Investment Management Services, Inc.
Jupiter Systems
KL Group Inc.
Massachusetts Institute of Technology
Metheus Corporation
Metro Link, Inc.
Object Management Group, Inc.
Open Software Foundation
Performance Awareness Corp.
Peritek Corp.
Petrotechnical Open Software Corp.
Point Technologies, Inc.
Shiman Associates, Inc.
Smithsonian Astrophysical Observatory.
Software Development Corp.
SOUM Corporation
Spectragraphics Corp.
Tech-Source, Inc.
TriTeal Corp.
White Pine Software, Inc.
World Wide Web Consortium.
The XFree86 Project, Inc.
X Inside, Inc.
.fi
